docs/cli-next/action.yml

```diff
@@ -57,27 +53,22 @@ runs:
       run: |
         cd current-repo
         make .bin/clidoc
-        if [[ "${{ inputs.arg }}" == "" ]]; then
-          .bin/clidoc ../docs/${{ inputs.output-dir }}
-        else
-          .bin/clidoc ${{ inputs.arg }} ../docs/${{ inputs.output-dir }}
-        fi
+        .bin/clidoc ${{ inputs.arg }} ../docs/${{ inputs.output-dir }}
 
-    - env:
-        GITHUB_TOKEN: ${{ inputs.token || github.token }}
-      if: ${{ github.ref_name == 'master' || github.ref_type == 'tag' }}
+    - uses: ory/ci/prettier@master
+      with:
+        dir: docs
+        action: write
+
+    - name: Push docs
       shell: bash
       run: |
         cd docs
-        git add -A
-        git commit -a -m "autogen(docs): generate cli docs" || true
-    - if: ${{ github.ref_name == 'master' || github.ref_type == 'tag' }}
```

Comment:
...and ran some git stuff twice. Was one of this intended for PRs and the other for master?
Regardless, this second git stuff step did not commit but only push, which also made no sense really.
My proposal is to just do everything in one step.